app/models/item.rb

```diff
@@ -37,4 +37,9 @@ def recent_costs
     repairs.where("service_date > ?", 6.months.ago).pluck(:cost).compact.inject(:+) || 0
   end
 
+  def location_name
+    location.name if location
+    "Unknown"
```

Comment:
Doesn't this ALWAYS return "Unknown" ?!

did you mean `return location.name if location` on the line above?